galoy-client/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "galoy-client"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+futures = "0.3.23"
+graphql_client = {version = "0.11.0", features = ["reqwest-blocking", "reqwest"]}
```

Comment:
Why blocking?

Reply:
Please disregard. I had initially implemented a blocking `GaloyClient::connect()` method but refactored it to be async because blocking calls panic in an async runtime. Feature has been removed.